Library/Power Ups/Power Ups Advantages.adq

```diff
@@ -9712,234 +9712,6 @@
 				"Talent"
 			]
 		},
-		{
-			"type": "advantage",
-			"id": "916c9bae-7bad-4225-8f7a-046665599ed4",
-			"name": "Talent (Chi Talent)",
```

Comment:
I noticed that this version didn't have the superfluous "Reaction Bonus: None" note... which made me then realize that all the other Talents that actually *do* have a reaction bonus were either done before you could actually specify reaction bonuses in GCS, or were just done wrong. Any chance you'd be willing to go through and adjust them as part of this PR?

Reply:
Sure, I'll take a look.

Reply:
I changed reaction text to make it clear to whom it applies.
I'm not sure if there is an easy way to fix conditional bonuses without manually rewording every single one of them, and in my opinion it's too much effort when "+1: Bonus to <thing>" is reasonably clear even if a bit unnatural. Btw, mangled Survivor text was probably someone's attempt to shorthand the full description.